addons/lunch/models/lunch_alert.py

```diff
@@ -14,7 +14,7 @@ class LunchAlert(models.Model):
     _description = 'Lunch Alert'
     _rec_name = 'message'
 
-    message = fields.Html('Message', required=True)
+    message = fields.Text('Message', required=True)
```

Comment:
We need to change this because it is set as the _rec_name but if we add a new field, we can keep the html field which was a requirement in the main task.
```python
name = fields.Char('Name', required=True)
```